src/System.Memory/tests/ReadOnlySpan/BinarySearch.cs

```diff
@@ -0,0 +1,190 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.SpanTests
+{
+    public static partial class ReadOnlySpanTests
+    {
+        public static readonly TheoryData<(uint[] Array, uint Value, int ExpectedIndex)> UIntCases =
```

Comment:
nit: camelCase the parameters, variable name, and append `s_` to the var name?
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md#c-coding-style

Reply:
@ahsonkhan camelCase the parameters? Aren't value tuple parameter name equal to public property names?

Reply:
Sometimes we treat them as public fields and other times we treat them as an argument list. I still like the parallelism between argument lists and tuples but I'm also curious what the guideline would be.

Reply:
@jnm2 yeah and I understand this argument, but to me they just really seem like public fields, they define a data structure. Anyway, hoping a guideline can clear this up. 😄